compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -156,7 +156,7 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         withDefaultPos(tpd.Import(expr, selectors))
       def copy(original: Tree)(expr: Term, selectors: List[Selector]): Import =
         tpd.cpy.Import(original)(expr, selectors)
-      def unapply(tree: Import): Option[(Term, List[Selector])] =
+      def unapply(tree: Import): Some[(Term, List[Selector])] =
         Some((tree.expr, tree.selectors))
```

Comment:
The return type can be just `(Term, List[Selector])`, it is also irrefutable -- product matches are always irrefutable. The same for the other changes.

Reply:
I was just investigating that possibility :)
I will update it.

Reply:
This also caught some extra bugs